src/structuralFunctionality/a11yUtils.ts

```diff
@@ -43,6 +42,45 @@ export function getMapDescription(
   }
   return config?.mapA11yDesc || appitem || mapItem;
 }
+
+export function setMapDescription(view: __esri.MapView | __esri.SceneView, mapA11yDesc: string, sanitizerInstance: any): void {
```

Comment:
Could you add a type instead of any. This is what I use: `interface ISanitizer {
  sanitize: (arg0: string) => string;
}`

Reply:
Done